packages/back-end/src/app.ts

```diff
@@ -227,6 +228,18 @@ app.use(
   apiRouter
 );
 
+// SCIM API routes (no JWT or CORS)
+app.use(
+  "/scim/v2",
+  bodyParser.json({
+    type: "application/scim+json",
+  }),
```

Comment:
This will make sure we get the body as a JSON and don't have to parse within each endpoint which was causing an error